pnpm-workspace.yaml

```diff
@@ -3,3 +3,7 @@ packages:
   - "workerd"
   - "wrangler-*"
   - "report"
+
+catalog:
+  wrangler: ^3.78.8
+  workerd: ^1.20240924.0
```

Comment:
aha!!!! I was wondering how this worked! this is awesome! thanks for doing this.

I only today learned about https://pnpm.io/catalogs — great find!